__tests__/app.test.js

```diff
@@ -59,42 +59,78 @@ describe("GET /api/topics", () => {
 describe("GET /api/articles/:article_id", () => {
   test("200: return object with correct properties and ID", () => {
     return request(app)
-    .get("/api/articles/1")
-    .expect(200)
-    .then(({body}) =>{
-      const {article_id} = body;
-      expect(typeof body).toBe('object')
-      expect(Array.isArray(body)).toBe(false)
-      expect(body).toHaveProperty("author")
-      expect(body).toHaveProperty("title")
-      expect(body).toHaveProperty("article_id")
-      expect(body).toHaveProperty('body')
-      expect(body).toHaveProperty('topic')
-      expect(body).toHaveProperty('created_at')
-      expect(body).toHaveProperty('votes')
-      expect(body).toHaveProperty('article_img_url')
-      expect(article_id).toEqual(1)
-    })
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article_id } = body;
+        expect(typeof body).toBe('object')
+        expect(Array.isArray(body)).toBe(false)
+        expect(body).toHaveProperty("author")
+        expect(body).toHaveProperty("title")
+        expect(body).toHaveProperty("article_id")
+        expect(body).toHaveProperty('body')
+        expect(body).toHaveProperty('topic')
+        expect(body).toHaveProperty('created_at')
+        expect(body).toHaveProperty('votes')
+        expect(body).toHaveProperty('article_img_url')
+        expect(article_id).toEqual(1)
+      })
```

Comment:
You are expecting article 1, what values are you expecting on article 1? You own the database and could find out.